assets/src/scripts/components/hydrograph/time-series-graph.js

```diff
@@ -0,0 +1,280 @@
+import { extent } from 'd3-array';
+import { line as d3Line, curveStepAfter } from 'd3-shape';
+import {createTooltipFocus, createTooltipText} from './tooltip';
```

Comment:
Let's clean up the import ordering. Put a space between library level imports (the d3-*) and then group together imports that are from the same directory, and put the imports in that group in alphabetical order. Makes it easier to find things.